src/components/app/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './App.css';
+import Login from '../login/Login';
+import { Route, Switch } from 'react-router-dom';
+
+const App = () => {
+  return (
+    <div className="App">
+      <Switch>
+        <Route path="/" component={Login} />
```

Comment:
why is the root path the login page? shouldn't this be `/login`

Reply:
Will change